README.md

```diff
@@ -51,7 +51,7 @@ Alternatively, skip all the steps by using [![Gitpod Ready-to-Code](https://img.
 
 - Contributions make the open source community such an amazing place to learn, inspire, and create.
 - Any contributions you make are **greatly appreciated**.
-- Check out our [contribution guidelines](https://github.com/EddieHubCommunity/LinkFree/blob/main/Contributing.md) for more information.
+- Check out our [contribution guidelines](/CONTRIBUTING.md) for more information.
```

Comment:
URLs are not case-sensitive, but if we still need that, I'd recommend a absolute URL over an relative one

```suggestion
- Check out our [contribution guidelines](https://github.com/EddieHubCommunity/LinkFree/blob/main/CONTRIBUTING.md) for more information.
```

Reply:
One reason of this PR was to make the URL short, even with just /CONTRIBUTING.MD it works well, so I don't see a need in this @krshdev.